v2/tests/common.go

```diff
@@ -97,10 +96,9 @@ func assertWorkerPhase(
 	}
 }
 
-// nolint: golint
 func assertJobPhase(
 	t *testing.T,
-	ctx context.Context,
+	ctx context.Context, // nolint: revive
```

Comment:
Curious as to the (ignored) linter issue here?

Reply:
Neither golint not revive like it when a function takes a `context.Context` as an arg and it's not the _first_ arg in the list. Generally, I think that's a good rule. For this case, I think `*testing.T` is the more appropriate thing to come first in the argument list.

Note that the `// nolint: golint` previously on line 100 was also meant to bypass this same rule. I've just moved it down a few lines to make this exception a little closer to the broken rule.